new-dti-website/components/course/DDProjects.tsx

```diff
@@ -35,6 +35,7 @@ export default function DDProjects({ title, description, imageSrc }: DDProjectsP
         isOpen ? 'bg-red-500' : 'bg-white'
       } w-full max-w-8xl rounded-xl drop-shadow-sm px-10 py-8 border-1 border-[#E4E4E4]`}
       onClick={toggleCard}
+      aria-label="toggle card"
```

Comment:
could we possibly make the alt text dynamic based on what the project is actually called?

ie: "Expand todo list app student project" or "Minimize todo list app student project"